language/move-prover/tests/sources/functional/ModifiesTest.move

```diff
@@ -0,0 +1,83 @@
+address 0x0 {
```

Comment:
You can put them both into one file. Via the `incorrect` convention, this is at least how we do it normally in this testsuite.